mteb/abstasks/Audio/AbsTaskZeroshotClassificaiton.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from datasets import Dataset
+
+from ...encoder_interface import Encoder
+from ...evaluation.evaluators import AudioZeroshotClassificationEvaluator
+from ..AbsTask import AbsTask, ScoresDict
+
+logger = logging.getLogger(__name__)
+
+
+class AbsTaskZeroshotClassification(AbsTask):
+    """Abstract class for ZeroshotClassification tasks
+    The similarity between an image and candidate text prompts, such as this is a .
```

Comment:
Can you continue docstring?
```suggestion
    The similarity between an image and candidate text prompts, such as this is a ....
```